template/fullstack/base-sessions/client/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import routes from "./config/routes";
+
+function App() {
+  return (
+    <div className="App">
+      <Routes>
+        {routes({}).map((route) => (
+          <Route key={route.path} path={route.path} element={route.element} />
+        ))}
+      </Routes>
+    </div>
+  );
+}
+
+export default App;
```

Comment:
I'd suggest renaming JSX files in this client folder from `.js` to `.jsx` for consistency 👍🏻

Reply:
This template is going to be removed for now, so i wont make that change, but nice catch